static/Dockerfile.s390x_gomplate

```diff
@@ -1,8 +1,8 @@
-FROM s390x/ubuntu:18.04
+FROM s390x/ubuntu:20.04
```

Comment:
**question:** why not also build gomplate on 21.04?

Reply:
forgot, will change